README.md

```diff
@@ -104,11 +106,17 @@ Create report from coredump:
 
     $ casr-core -f casr/tests/casr_tests/bin/core.test_destAv -e casr/tests/casr_tests/bin/test_destAv -o destAv.casrep
 
-Create report from sanitizers output:
+Create report from AddressSanitizer output:
 
     $ clang++ -fsanitize=address -O0 -g casr/tests/casr_tests/test_asan_df.cpp -o test_asan_df
     $ casr-san -o asan.casrep -- ./test_asan_df
 
+Create report from UndefinedBehaviorSanitizer output:
+
+    $ clang++ -fsanitize=undefined -O0 -g casr/tests/casr_tests/ubsan/test_ubsan.cpp -o test_ubsan
+    $ casr-ubsan -i casr/tests/casr_tests/ubsan/input1/input -o output -- ./test_ubsan @@
```

Comment:
casr/tests/casr_tests/ubsan/input1/input -> casr/tests/casr_tests/ubsan/input1

the same in usage.md